spec/terrestrial/lazy_collection_spec.rb

```diff
@@ -7,7 +7,7 @@
     Terrestrial::LazyCollection.new(
       database_enum,
       loader,
-      mapper,
+      queries,
```

Comment:
This was actually a bit naughty of me. This argument was never a `mapper` it's actually a hash of predefined queries like `ActiveRecord` scopes. It's so far only tested at the integration level, which meant having an incorrect third argument in the test didn't break anything. Separate commit? @mikejustdoit